private/buf/cmd/buf/command/alpha/registry/token/tokenget/tokenget.go

```diff
@@ -87,6 +88,10 @@ func run(
 	if remote == "" {
 		return appcmd.NewInvalidArgumentError("you must specify a remote module")
 	}
+	parts := strings.Split(remote, "/")
+	if len(parts) > 1 {
+		return appcmd.NewInvalidArgumentError("remote address cannot contain subdirectories")
```

Comment:
Maybe just me, but I prefer the term path over subdirectory.

`invalid remote address, must not contain any paths`

If you wanted to get fancy can parse the path and do something like:

`invalid remote address, must not contain any paths. Try removing "/foo/bar" from the address.`

Reply:
Challenge accepted.  Added a validation file to hold this too.